go/test/endtoend/cluster/cluster_util.go

```diff
@@ -498,3 +499,47 @@ func DialVTGate(ctx context.Context, name, addr, username, password string) (*vt
 	vtgateconn.RegisterDialer(dialerName, dialerFunc)
 	return vtgateconn.DialProtocol(ctx, dialerName, addr)
 }
+
+// PrintFiles prints the files that are asked for. If no file is specified, all the files are printed.
+func PrintFiles(t *testing.T, dir string, files ...string) {
+	var directories []string
+	directories = append(directories, dir)
```

Comment:
Won't this always be one directory? If an empty string is passed, we can just return.